tools/ci_build/exclude_unused_ops.py

```diff
@@ -276,6 +323,16 @@ def process_lines(lines, offset):
                 file_to_write.write(line)
                 line_offset += 1
 
+    # make sure we have an unversioned last entry for each operator unless it's deprecated
+    for entry in last_op_registrations.items():
+        key, value = entry
+        opset_from, opset_to = value
+
+        deprecated = key in deprecated_ops and opset_to == deprecated_ops[key] - 1
```

Comment:
>deprecated_ops[key] - 1 [](start = 59, length = 23)

If we do this calculation every time and there is no other dependency on deprecated_ops[key], why not we just save the targeted value there for convenience?

Reply:
We do this once after processing all registrations. Not sure what you mean by 'save the targeted value there'.

---
In reply to: [523938771](https://github.com/microsoft/onnxruntime/pull/5817#discussion_r523938771) [](ancestors = 523938771)